app/code/core/Mage/Core/functions.php

```diff
@@ -165,7 +165,7 @@ function mageCoreErrorHandler($errno, $errstr, $errfile, $errline)
         case E_USER_NOTICE:
             $errorMessage .= 'User Notice';
             break;
-        case E_STRICT:
+        case PHP_VERSION_ID < 80400 && E_STRICT:
```

Comment:
```suggestion
        case 2048: // E_STRICT prior to PHP8.4
```

Reply:
It is much better using the error value in this case.